File: src/utilities/sendMail.js
import * as MailComposer from 'expo-mail-composer';


const sendMail = async (subject) => {
    try {
        const isAvailable = await MailComposer.isAvailableAsync();
        if (isAvailable) {
            await MailComposer.composeAsync({
                recipients: ['team@lingovido.com'],
                subject,
            });
        } else {
            console.log('mail error')
        }
    } catch (error) {
        console.log('mail error', error)

    }

};

export default sendMail

File: lib/ultils/functions/send_mail.dart
import 'package:flutter_email_sender/flutter_email_sender.dart';

Future<void> sendMail(String subject) async {
  final Email email = Email(
    subject: subject,
    recipients: ['team@lingovido.com'],
    isHTML: false,
  );

  await FlutterEmailSender.send(email);
}
